### PR TITLE
add cpp solution to lc problem no:68

### PR DESCRIPTION
**Pull request by Dhoni77**
0068.Text Justification

### CODE REVIEW

File: solution/0000-0099/0068.Text Justification/README_EN.md [hunk 1]
```diff
@@ -86,10 +86,52 @@ Note that the second line is also left-justified becase it contains only one wor
 
 ```
 
-### **...**
+### **C++**
 
 ```
```
Comment (YangFong) [inline solution/0000-0099/0068.Text Justification/README_EN.md:91]: `cpp` tags are missing

Comment (Dhoni77) [reply solution/0000-0099/0068.Text Justification/README_EN.md:91]: As I am new here can you guide me where to add the tags ?
I added **C++** in line 89 and is that the tag you are mentioning?

**Comment by YangFong** [comment]
I found that I could not select that line, I wanted to say Markdown code block, need to add `cpp`

**Comment by Dhoni77** [comment]
> I found that I could not select that line, I wanted to say Markdown code block, need to add `cpp`

ok done!

**Comment by yanglbme** [comment]
Please create a Solution.cpp file and add the solution to it. And I'll close #740 @Dhoni77

**Comment by Dhoni77** [comment]
> Please create a Solution.cpp file and add the solution to it. And I'll close #740 @Dhoni77

Ok doing it now

**Comment by yanglbme** [comment]
Thanks for contributing! I'll format your code before merging into main branch. @Dhoni77

**Review by yanglbme — APPROVED** [review]

**Comment by Dhoni77** [comment]
Thank you @yanglbme and @YangFong.
This is my first contribution and I'll try to contribute more😍

**Pull request merged**